test/routes/permitHolder/permitHolderNameAndDateOfBirth.route.test.js

```diff
@@ -121,44 +79,71 @@ const checkPageElements = async (request, expectedFirstName = '', expectedLastNa
   Code.expect(element.nodeValue).to.equal('Continue')
 }
 
-const checkValidationErrors = async (field, expectedErrors) => {
-  const doc = await GeneralTestHelper.getDoc(postRequest)
-
-  let element
-
-  for (let i = 0; i < expectedErrors.length; i++) {
-    // Panel summary error item
-    element = doc.getElementById(`error-summary-list-item-${i}`).firstChild
-    Code.expect(element.nodeValue).to.equal(expectedErrors[i])
-
-    // Field error
-    Code.expect(doc.getElementById(field).getAttribute('class')).contains('form-control-error')
-    element = doc.getElementById(`${field}-error`).childNodes[i].firstChild
-    Code.expect(element.nodeValue).to.equal(expectedErrors[i])
-  }
-}
-
 lab.experiment('Permit Holder Name page tests:', () => {
+  let getRequest
+
   new GeneralTestHelper({ lab, routePath }).test()
 
+  lab.beforeEach(() => {
+    getRequest = {
+      method: 'GET',
+      url: routePath,
+      headers: {}
+    }
+  })
+
   lab.experiment('GET:', () => {
     lab.test(`GET ${routePath} returns the permit holder name page correctly when it is a new application`, async () => {
       // No current permit holder
-      fakeContactDetail = {}
+      sinon.stub(ContactDetail, 'get').value(() => undefined)
```

Comment:
Rather than stubbing outside the sandbox, which causes side-effects, it would be better to re-use a reference to the already stubbed method or just clear the value being returned by the mocks